ezyhttp-client/src/main/java/com/tvd12/ezyhttp/client/request/RequestEntity.java

```diff
@@ -4,117 +4,71 @@
 import com.tvd12.ezyhttp.core.data.MultiValueMap;
 
 import lombok.Getter;
 
 @Getter
-public class RequestEntity {
+public class RequestEntity<T> {
 
-	protected final Object body;
+	protected final T body;
 	protected final MultiValueMap headers;
-	
-	public RequestEntity(MultiValueMap headers, Object body) {
-		this.body = body;
-		this.headers = headers;
-	}
-	
-	public RequestEntity(Map<String, List<String>> headers, Object body) {
-		this(headers != null ? new MultiValueMap(headers) : null, body);
-	}
-	
-	public static Builder of(Object body) {
-		return builder().body(body);
+
+	public RequestEntity() {
+		this.body = null;
+		this.headers = null;
 	}
-	
-	public static RequestEntity body(Object body) {
-		return new RequestEntity((MultiValueMap)null, body);
+
+	public RequestEntity(T body) {
+		this.body = body;
+		this.headers = null;
 	}
-	
-	@SuppressWarnings("unchecked")
-	public <T> T getBody() {
-		return (T)body;
+	public RequestEntity(MultiValueMap headers, T body) {
+		this.body = body;
+		this.headers = headers;
 	}
-	
-	public String getHeader(String name) {
-		if(headers == null)
-			return null;
-		String value = headers.getValue(name);
-		return value;
+
+	public static <T> Builder<T> of(T body) {
+		return new Builder<T>().body(body);
 	}
-	
-	public String getContentType() {
-		if(headers == null)
-			return ContentTypes.APPLICATION_JSON;
-		String contentType = headers.getValue(Headers.CONTENT_TYPE, ContentTypes.APPLICATION_JSON);
-		return contentType;
+
+	public T getBody() {
+		return this.body;
 	}
-	
+
 	@Override
 	public String toString() {
-		return new StringBuilder()
-				.append("RequestEntity(")
-				.append("headers: ").append(headers).append(", ")
-				.append("body: ").append(body != null ? body.getClass().getSimpleName() : "null")
-				.append(")")
-				.toString();
-	}
-	
-	public static Builder builder() {
-		return new Builder();
+		return "RequestEntity(" +
+				"headers: " + headers + ", " +
+				"body: " + (body != null ? body.getClass().getSimpleName() : "null") +
+				")";
 	}
 	
-	public static class Builder implements EzyBuilder<RequestEntity> {
+	public static class Builder<T> implements EzyBuilder<RequestEntity<T>> {
 
-		protected Object body;
+		protected T body;
 		protected Map<String, List<String>> headers;
 		
-		public Builder body(Object body) {
+		public Builder<T> body(T body) {
 			if(body != null) {
-				if(body instanceof MultiValueMap)
-					this.body = ((MultiValueMap)body).toMap();
-				else
-					this.body = body;
+				this.body = body;
 			}
 			return this;
 		}
 		
-		public Builder header(String name, String value) {
+		public Builder<T> header(String name, String value) {
 			if(this.headers == null)
 				this.headers = new HashMap<>();
-			List<String> values = headers.get(name);
-			if(values == null) {
-				values = new ArrayList<>();
-				headers.put(name, values);
-			}
+			List<String> values = headers.computeIfAbsent(name, k -> new ArrayList<>());
```

Comment:
LGTM, but you should declare (in Builder class) to make more meanning:

```java
protected final Function NEW_HEADER_VALUES_FUNC = k -> new ArrayList<>();
```

and use like bellow:

```java
List<String> values = headers.computeIfAbsent(name, NEW_HEADER_VALUES_FUNC);
```